### ISSUE TITLE
Many Errors , unable yo build to Android Until i delete IOS library 

### ISSUE THREAD

**Issue by nitroman123** [CesiumGS/cesium-unity#514]
![image](https://github.com/user-attachments/assets/3d8912ee-0b14-4099-8458-b703feee1e78)

unity 2021.3.44

**Comment by kring** [comment]
I'm not sure what might be happening there. It's not obvious those errors have anything to do with iOS. It is common to see some iOS-related _warnings_, but they shouldn't cause any harm. One thing that can cause the sort of errors in your screenshot is attempting to build for 32-bit Android. Only 64-bit Android (ARMv8) is supported.

If you think there's something we need to change in Cesium for Unity, please walk us through what you did and what you saw so that we can reproduce the problem ourselves.

**Comment by nitroman123** [comment]
this ones , are errors , but my workaround in order to build il2cpp was to delete ios library... mono for some reason it could build in mono with library , not sure what steps you need to take...

**Comment by kring** [comment]
You're saying it does build when targetting Mono, without deleting the iOS binaries? That's pretty surprising, because I believe Mono only runs on ARMv7 on Android, and that platform isn't supported at all.

**Comment by nitroman123** [comment]
yes i don't know why... yes i found first this that i ve made that discovering first and then i used the logs and i decided to delete the iOS library and it works...

**Comment by nitroman123** [comment]
The process of deleting and re-import the library it's not very viable as
you understand everytime  you switch from ios to andriod and vise versa...
can you reproduce with unity 2021.3.44f1 ?

**Comment by nitroman123** [comment]
not the propper way and how it suppose to be

**Comment by Miquel-scala** [comment]
Hi ,problem is that iOS plugin content is set to any platform, but it should only be iOS only
